__tests__/resources/openapi-with-internal.yml

```diff
@@ -0,0 +1,90 @@
+openapi: 3.0.1
+info:
+  title: My API
+  version: 1.0.0
+paths:
+  /pets:
+    get:
+      operationId: getPets
+      x-internal: true
+      responses:
+        '200':
+          $ref: '#/components/responses/ListPetsRes'
+    post:
+      operationId: createPet
+      requestBody:
+        description: Pet object to create
+        content:
+          application/json: {}
+      responses:
+        '201':
+          $ref: '#/components/responses/PetRes'
+  '/pets/{id}':
+    get:
+      operationId: getPetById
+      responses:
+        '200':
+          $ref: '#/components/responses/PetRes'
+      parameters:
+      - name: id
+        in: path
+        required: true
+        schema:
+          type: integer
+      x-internal: true
+components:
+  schemas:
+    Pet:
+      type: object
+      properties:
+        id:
+          type: integer
+          minimum: 1
+        name:
+          type: string
+          example: Odie
+  responses:
+    ListPetsRes:
+      description: ok
```

Comment:
can we add x-internal here? since the path using this is internal.
and show the PoC

Reply:
hmmm. that's awesome 💪🏼